packages/swingset-theme-hashicorp/tailwind.config.cjs

```diff
@@ -7,11 +9,11 @@ module.exports = {
   ],
   theme: {
     extend: {
-      colors: {},
       textColor: {
-        'foreground-primary': '#3b3d45',
-        'foreground-faint': '#656a76',
-        'foreground-action': '#1060ff',
+        dark: colors.slate[700],
+        primary: colors.slate[600],
+        faint: colors.gray[500],
+        action: '#1060ff',
```

Comment:
When we pass a custom class into the `textColor` sub-object. Tailwind exposes the style as `text-{className}`.

So currently, the class name is `ss-text-foreground-primary`. Foreground and text is a redundancy.